views/player/partials/giveItemButton.ejs

```diff
@@ -96,7 +97,7 @@
       let quality = $('#player-give-item-input-quality').val()
 
       if (!(amount == '')) {
```

Comment:
This one should be quality as well right?

Reply:
I thought that was "If the amount isn't blank, then check the quality - otherwise, complain about the amount being blank (in the next check)"

Reply:
Oh yeah, you're right!